client/src/components/ChooserWidget/index.js

```diff
@@ -0,0 +1,97 @@
+import $ from 'jquery';
```

Comment:
I know there is not a clear guideline yet but not sure this should be in the `components` folder.

Again, this should probably be in the `includes` folder for now as this is not a React component.

Also, just curious if there is a possibility to make this typescript (I am happy to help with this).

Reply:
FWIW `components/StreamField` sets a precedent of breaking that rule (which is not to say it's a good thing to break it again)... I chose `components` here as it seemed more conceptually component-y than the other code (being an identifiable element on the page with JS behaviour). Happy to go with whatever conventions we establish though.